lib/private/DB/QueryBuilder/QuoteHelper.php

```diff
@@ -57,6 +57,10 @@ public function quoteColumnName($string) {
 		if ($string === null || $string === 'null' || $string === '*') {
 			return $string;
 		}
+		
+		if ($string instanceof \Doctrine\DBAL\Schema\Column) {
```

Comment:
```suggestion
		if ($string instanceof Column) {
```

And Column could be imported on top:
``` php
use Doctrine\DBAL\Schema\Column;
```